phpstan.neon

```diff
@@ -0,0 +1,8 @@
+parameters:
```

Comment:
Why not phpstan.neon.dist?

Reply:
Missed it, making a new PR

Reply:
https://github.com/Roave/BackwardCompatibilityCheck/pull/99